### PR TITLE
Feature/todos 20201001

### PR DESCRIPTION
**Pull request by rewhowe**
Finish a couple TODOs (updating documentation, adding more tests and validation, etc), added documentation for "null" (will be implemented quickly next)

### CODE REVIEW

File: documentation/manual_jp.md [hunk 1]
```diff
@@ -222,9 +241,26 @@
 
 勿論、これらもひらがなで書けます。
 
-### ELSE IF and ELSE (TODO jp)
+### 「else if」と「else」
+
+「もし」の条件分岐の後に、その文と同じインデントで並んだ「else if」または「if」という命令も書けます。
```
Comment (rewhowe) [inline documentation/manual_jp.md:246]: Fix here too. And stop using code editors to edit markdown in Japanese...
